src/renderer/mod.rs

```diff
@@ -60,7 +60,16 @@ impl Renderer {
         }
     }
 
+    #[cfg(not(feature = "testing-colors"))]
     /// Default terminal styling
+    ///
+    /// # Note
+    /// Some colors are OS dependant, meaning that the colors will look
+    /// different on different operating systems, making testing colored output
+    /// difficult.
+    /// If you are testing the colored output, you can use the
+    /// [`testing-colors` feature](crate#features) to enable colors that are OS
+    /// independent.
```

Comment:
I'd explicitly call this out as a `dev-dependency`, maybe showing the command `cargo add annotate-snippets --dev --feature testing-colors`